src/couch/src/couch_db.erl

```diff
@@ -397,10 +376,7 @@ get_full_doc_info(Db, Id) ->
     Result.
 
 get_full_doc_infos(Db, Ids) ->
```

Comment:
Let's remove or update [this comment]
(https://github.com/apache/couchdb/blob/7162a24ba6b6341a9c015a5fc6b6f4d4306ba432/src/couch/src/couch_db.erl#L373), 5 lines above since now we return either `#full_doc_info` or `not_found`